shelljs/shelljs.d.ts

```diff
@@ -511,6 +511,18 @@ declare module "shelljs"
      */
     export function error(): string;
 
+
+    export function touch(...files: string[]): void;
+    export function touch(files: string[]): void;
+
+    export function touch(options: string, ...files: string[]): void;
+    export function touch(options: string, files: string[]): void;
+    
+    export function touch(options: Object, ...files: string[]): void;
+    export function touch(options: Object, files: string[]): void;
```

Comment:
Well, I just noticed that the options strings can be combined into one, eg `-ac`. I think the best way to model this is to just use the raw strings/objects that you proposed earlier.
Sorry for the confusion.